integration/apps/rspec/app/datadog.rb

```diff
@@ -1,10 +1,20 @@
 require 'datadog/demo_env'
-require 'ddtrace'
+require 'datadog/ci'
```

Comment:
If I'm looking at this right, this test app is the only one right now that validates that customers can just use `require 'datadog/ci'` to use CI, instead of having to use `require 'ddtrace'`.

This seems a pretty important job. So I suggest adding a comment here stating that, because otherwise this require just blends into the background whereas it's one of the most important things being validated here.